src/urbit/mar/chess/archive.hoon

```diff
@@ -0,0 +1,19 @@
+/-  *chess
+/+  chess
+=,  format
+|_  archive=chess-archive
+++  grab
+  |%
+  ++  noun  chess-archive
+  --
+++  grow
+  |%
+  ++  noun  archive
+  ++  json
+    %-  frond:enjs
+    :-  'localArchive'
+    :-  %a  (turn (tap:arch-orm archive) archive-json:chess)
```

Comment:
collapsing things into a list might be causing the ordered map problems, like not ordering things.

Reply:
If the `tap` function is resulting in incorrect order from an ordered map, that's a pretty big bug...

Reply:
I found my user error.

In [app/chess](https://github.com/thecommons-urbit/chess/pull/112/files#diff-b795e051f49f72ee01409845fe30b201053ae7e7e4e6a6712aade843f305c196) `~(put by archive)` wasn't replaced with `(put:arch-orm archive )` yet.

strange, but good, that the map arms work for mop. 

I also expected the archive to reorganize itself when I tweaked the mop or even the archive mark, but apparently the way it's `put` in "locks" things in place.